block-storage/src/main/scala/coop/rchain/blockstorage/util/io/package.scala

```diff
@@ -4,36 +4,41 @@ import java.io.{EOFException, IOException}
 import java.nio.file._
 import java.util.stream.Collectors
 
-import cats.data.EitherT
 import cats.implicits._
 import cats.effect.Sync
-import coop.rchain.blockstorage.util.io.IOError.{IOErr, IOErrT}
+import coop.rchain.blockstorage.util.io.IOError.RaiseIOError
 
 import scala.collection.JavaConverters._
 
 package object io {
-  private[io] def handleIo[F[_]: Sync, A](
+  private[io] def handleIo[F[_]: Sync: RaiseIOError, A](
```

Comment:
Since `RaiseIOError` delegates to `Sync` anyway you should be able to have just `Sync` requirement and materialize `RaiseIOError`.

Reply:
My commit is irrelevant in the light of @itegulov answer to previous one.